sqlite_minutils/db.py

```diff
@@ -3652,10 +3652,13 @@ def jsonify_if_needed(value):
 
 
 def resolve_extracts(
-    extracts: Optional[Union[Dict[str, str], List[str], Tuple[str]]]
+    extracts: Optional[Union[Dict[str, str], List[str], Tuple[str], Tuple[Tuple[str]]]]
```

Comment:
New type option needs to be handled because it is a regression caused by the new hashing function.